### ISSUE TITLE
add bill tracker and bill progress website

### ISSUE THREAD

**Issue by playbass** [playbass/UtahLegislatureInfo#1]
download current bills in both houses and mark their progress. list sponsors and emails/phone #

**Comment by todrobbins** [comment]
@playbass are you familiar with the OpenStates project? They're already doing a lot of this at the state level here: https://openstates.org/ut/

**Comment by playbass** [comment]
Yeah, I'm using their API actually through python sunlight foundation library. I was thinking of using this data and creating a more heads up display of what bills are in what stage of the legislative process but I haven't look at the OpenStates in several months they may have something closer to what I want out of the box.

**Comment by todrobbins** [comment]
v2 of the API is impending. Let me know if you want an invite to the Open States Slack!

**Comment by playbass** [comment]
Good to hear! Yes, please, I'll take an invite. I wouldn't mind contributing there if I can as well.

**Comment by todrobbins** [comment]
Send [me](todd.d.robbins@gmail.com) your email address and I'll send you an invite.
